external/wycheproof/README.md

```diff
@@ -0,0 +1,31 @@
+# External Wycheproof Tests
+
+Wycheproof tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Wycheproof test material is pulled from the [wycheproof](https://github.com/google/wycheproof) repository.
+
+## Running External tests locally
+
+To run any AQA tests locally, you follow the same pattern:
+
+0. Ensure your test machine is set up with [test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md).  For external tests, you do need Docker installed.
+
+1. Download/unpack the SDK you want to your test machine
+
+2. `export TEST_JDK_HOME=</pathToWhereYouInstalledSDK>` 
+
+3. `git clone https://github.com/AdoptOpenJDK/openjdk-tests.git`
+
+4. `cd openjdk-tests`
+
+5. `./get.sh`
+
+6. `cd TKG`
+
+7. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (example: `export BUILD_LIST=external/wycheproof` and `export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`
+
+8. `make wycheproof`        (This fetches test material and compiles it, based on build.xml files in the test directories)
+
+9. `make wycheproof_test`   (When you defined BUILD_LIST to point to a directory in [openjdk-tests/external](https://github.com/AdoptOpenJDK/openjdk-tests/tree/master/external), then this is a testCaseName from the playlist.xml file within the directory you chose)
+
+When [running these from the command-line](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#local-testing-via-make-targets-on-the-commandline), these tests are grouped under a make target called 'external', so 'make external' would run the entire set of tests found in the openjdk-tests/external directory. This is unadvisable! Limit what you compile and run, BUILD_LIST=external/<someSubDirectory>, and TARGET=<testCaseNameFromSubdirPlaylist>
```

Comment:
The last sentence you will need to add `` before < and after >, to make the context visible. That is BUILD_LIST=external/`<someSubDirectory>`, and TARGET=`<testCaseNameFromSubdirPlaylist>`.

And I would suggest to update as BUILD_LIST=external/wycheproof, and TARGET=`<testCaseNameFromPlaylistUnderSubdirwycheproof>`

Reply:
Thanks @sophia-guo for pointing this out, making the changes ASAP.

Reply:
Should the wycheproof in this sentence be in angle brackets as well like you did for "<testCaseNameFromPlaylistUnderDirwycheproof>"  BUILD_LIST=external/wycheproof, and TARGET=<testCaseNameFromPlaylistUnderDirwycheproof>

Reply:
As it is a specific subdirectory so i think no need to put the angle brackets.

Reply:
Alright, thanks.

Reply:
Hi @sophia-guo, The required changes has been made, kindly review.